SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/AbstractLBStrategy.cpp

```diff
@@ -14,7 +14,3 @@ void UAbstractLBStrategy::SetLocalVirtualWorkerId(uint32 InLocalVirtualWorkerId)
 {
```

Comment:
Can we change the type of the argument in this function from `uint32` to `VirtualWorkerId` to match the header?

Reply:
done